docs/sources/static/set-up/install-agent-docker.md

```diff
@@ -1,61 +1,62 @@
 ---
-title: Run static mode on Docker
+title: Docker
 weight: 110
 aliases:
 - ../../set-up/install-agent-docker/
 ---
 
-## Run static mode on Docker
+# Install Grafana Agent on Docker
 
-Install Grafana Agent and get it up and running on Docker.
+Grafana Agent is available as a Docker container image on the following platforms:
 
-### Before you begin
+* [Linux][]
+* [Windows][]
 
- - Ensure that you have Docker installed.
- - Ensure that you have created a configuration file. In the case of Docker, you install and run the Grafana Agent with a single command. You therefore need to create a configuration file before running Grafana Agent on Docker. For more information on creating a configuration file, refer to [Create a configuration file]({{< relref "../configuration/create-config-file/" >}}).
+[Linux]: #run-a-linux-docker-container
+[Windows]: #run-a-windows-docker-container
 
-### Run Agent in a Linux Container
+## Before you begin
 
-1. Copy and paste the following commands into your command line.
-```
-docker run \
-  -v /tmp/agent:/etc/agent/data \
-  -v /path/to/config.yaml:/etc/agent/agent.yaml \
-  grafana/agent:v0.33.2
-```
+ - Ensure that [Docker][] is installed and running on your machine.
+ - Ensure that you have created a Grafana Agent [configuration file]({{< relref "../configuration/create-config-file/" >}}).
 
-2. Replace `/tmp/agent` with the folder you want to store WAL data in.
+[Docker]: https://docker.io
+## Run a Linux Docker container
 
-    WAL data is where metrics are stored before they are sent to Prometheus. Old WAL data is cleaned up every hour and is used for recovery if the process happens to crash.
+1. To install a Grafana Agent Docker container on Linux, run the following command in a terminal:
```

Comment:
nit about the terminology here: I think "installing on Docker" is misleading, because it might sound like we're going to be adding Grafana Agent to an existing Docker container.

I think "running on Docker" is more correct for this specific platform.